cads_adaptors/tools/convertors.py

```diff
@@ -5,19 +5,19 @@
 
 STANDARD_COMPRESSION_OPTIONS = {
     "default": {
-        "compression": "gzip",
-        "compression_opts": 9,
-        "shuffle": True,
+        "zlib": True,
+        "complevel": 1,
         "engine": "h5netcdf",
```

Comment:
@EddyCMWF Is this going to be the default compression for NetCDF files?
If yes, I think it's a pretty important breaking change. Python users must install `h5netcdf` to read data as the filter is not supported by `netcdf4`.

To double check, I tried to read the compressed files generated by the unit test and I get this:
```
E   RuntimeError: NetCDF: Filter error: undefined filter encountered
```

Reply:
It's still under discussion, so subject to change. Your point is a good one though, do you have a recommendation? I have been doing various tests of performance regarding the speed of writing and effect of the compression. Largely influenced by what Marco Cucchi did for us a few years ago.

FYI, there will be other breaking changes with the netCDF as cfgrib does things differently to grib_to_netcdf, e.g. handling of the time_dimensions

Reply:
https://confluence.ecmwf.int/display/CDS2/GRIB+to+netCDF+compression+efficiency

Reply:
Also, I don't get the error you report, can you send an MRE?

Reply:
mmm, I can't reproduce the error anymore. Not sure what happened.
It's possible it only occurs with specific versions of netcdf4/libnetcdf/h5netcdf (I updated my env this morning).
I suggest to add a check in the unit test to make sure that the conversion does not create issues (which is what I tried yesterday):
```python
ds_grib = xr.open_dataset(tmp_grib_file)
ds_netcdf = xr.open_mfdataset(netcdf_files, engine="netcdf4")
xr.testing.assert_equal(ds_grib, ds_netcdf)
```

I don't have access to the confluence page.
I think it would be safer to use netcdf4 by default if possible. I tried this and it worked well: `{"zlib": True, "complevel": 1, "engine": "netcdf4"}`

Reply:
ohhh I know what happened!
I tried the compression algo you implemented in your first commit, which was this:
```python
{
    "default": {
        "compression": "lzf",
        "engine": "h5netcdf",
    }
}
```
So the error does not occur with the latest version (which I guess makes my point of using the `netcdf4` engine by default to make sure the default is not using a `h5netcdf` specific compression).

Reply:
Ahh, okay, sorry for the confusion. After a bit more testing I saw that I got faster and smaller results with the zlib, which I was more familiar with too.
I will also try setting the default engine to netcdf4 to make extra certain that we get cmpatible files.

Reply:
Also, I will rename the PR WIP